StyleCop.Analyzers/StyleCop.Analyzers.Test/OrderingRules/SA1203UnitTests.cs

```diff
@@ -268,14 +272,14 @@ public class Foo
 
             var fixedTestCode = @"public class Foo
 {
+
+    public const string After2 = ""test"";
     private const string Before1 = ""test"";
 
     public const string Before2 = ""test"";
 
     private const string After1 = ""test"";
 
-    public const string After2 = ""test"";
-
     private int field1;
 
     public int between;
```

Comment:
:question: Is this reordering correct? I would assume that a lot of users will find it annoying if their constants get rearranged when the code fix is just moving a field below a set of constants.